sprint0/sprint0/Collision/EnemyEnemyCollisionHandler.cs

```diff
@@ -13,13 +13,7 @@ public EnemyEnemyCollisionHandler()
 
         public void HandleCollision(IEnemy enemy1, IEnemy enemy2, Direction side)
         {
-            if (!(enemy1 is ManhandlaLimb && enemy2 is Manhandla) && !(enemy2 is ManhandlaLimb && enemy1 is Manhandla)
-                 && !(enemy1 is Zol && enemy2 is Gel) && !(enemy2 is Zol && enemy1 is Gel) && !(enemy1 is Zol && enemy2 is Zol)
-                 && !(enemy2 is PatraMinion && enemy1 is Patra) && !(enemy1 is PatraMinion && enemy2 is Patra)
-                 && !(enemy1 is Gleeok && enemy2 is GleeokNeckPiece) && !(enemy2 is Gleeok && enemy1 is GleeokNeckPiece)
-                 && !(enemy1 is Gleeok && enemy2 is GleeokHead) && !(enemy2 is Gleeok && enemy1 is GleeokHead)
-                 && !(enemy1 is GleeokNeckPiece && enemy2 is GleeokHead) && !(enemy2 is GleeokNeckPiece && enemy1 is GleeokHead)
-                 &&!(enemy2 is GleeokNeckPiece && enemy1 is GleeokNeckPiece) && !(enemy2 is GleeokHead && enemy1 is GleeokHead))
+            if (enemy1.Type == EnemyType.None || enemy2.Type == EnemyType.None || enemy1.Type != enemy2.Type)
```

Comment:
should this be `enemy1.Type == EnemyType.None && enemy2.Type == EnemyType.None && enemy1.Type != enemy2.Type` ?
Since we want them both to not any of the other types and also they should be different types